include/zenoh-pico/protocol/core.h

```diff
@@ -211,6 +211,20 @@ _z_keyexpr_t _z_rname(const char *rname);
  */
 _z_keyexpr_t _z_rid_with_suffix(uint16_t rid, const char *suffix);
 
+/**
+ * QoS settings of zenoh message.
+ *
+ * Members:
+ *   _z_priority_t priority: Priority of the message.
+ *   _z_congestion_control_t congestion_control: Congestion control of the message.
+ *   bool express: If true, the message is not batched during transmission, in order to reduce latency.
+ */
+typedef struct {
+    z_priority_t priority;
+    z_congestion_control_t congestion_control;
+    bool express;
+} _z_qos_t;
+
```

Comment:
The comment and the structure do not agree on type visibility (`_z_priority_t` vs `z_priority_t`), also we use `_Bool` for booleans generally.

Reply:
fixed